.github/workflows/deployBlocker.yml

```diff
@@ -54,7 +54,7 @@ jobs:
               channel: '#deployer',
               attachments: [{
                 color: "#DB4545",
-                text: '💥 New E.cash Deploy Blocker: <${{ env.DEPLOY_BLOCKER_URL }}|${{ env.DEPLOY_BLOCKER_TITLE }}>',
+                text: '💥 New E.cash Deploy Blocker: <${{ env.DEPLOY_BLOCKER_URL }}|'+ `${{ env.DEPLOY_BLOCKER_TITLE }}`.replace(/(^'|'$)/gi, '').replace(/'\''/gi,'\'') + '>',
```

Comment:
Hmmm I don't think this is likely to work? This isn't a JS string, it's using the [expression syntax for GH actions](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions).

Reply:
Look at here https://action-slack.netlify.app/with#custom_payload example. We can use js. 
I checked the action code its is using `eval()`;

Reply:
>  We can use js

Whoa, that's a nice feature of action-slack.

Reply:
Yeah totally agree, despite the use of eval.

Reply:
Can you also show me what local testing you did for this?

Reply:
This one is untested apart from the js code which I tested so that it gives us the original string.
I am not too sure about the + usages here. It could be the problem. But I feel it will work based on these:
1. I tested Yaml syntax which is valid.
2. Github will pass the string as it is after replacing the placeholders.
3. Now eval should parse the value. So I tested if the string is valid js syntax. Which it is after Gh action placeholders replacements.

Reply:
Can you show me the tests you did for the JS replacement portion? Even just using a Node.js REPL and some sample strings to demonstrate how the replacement works.

Reply:
1. Used this for YAML validation.
2. for the eval part you can use this https://jsbin.com/roxenoyibu/edit?js,console to test out strings. 
3. You can use https://github.com/Expensify/Expensify.cash/pull/4049#discussion_r670514803 to get strings.